### ISSUE TITLE
OSGI support

### ISSUE THREAD

**Issue by richardwilly98** [richardwilly98/es-dms#25]
Check feasibility after supporting OSGI

**Comment by danilos** [comment]
yes I think OSGI would be an excellent way to go. Spring DM (Blueprint) or DS?

http://wiki.osgi.org/wiki/Declarative_Services
